cli/popper/commands/cmd_info.py

```diff
@@ -0,0 +1,60 @@
+import click
+import popper.utils as pu
+import requests
+import os
+from popper.cli import pass_context
+
+try:
+    FileNotFoundError
+except NameError:
+    FileNotFoundError = IOError
```

Comment:
is this what you intended to have on this file? If so, can you please add a comment to explain what this is doing?

Reply:
Yes. I have been trying to write (and test) code which is compatible with both python3.x and python2.x versions.
FileNotFoundError does not exists in python2.x which makes use of "IOError" for such things.